### PR TITLE
Update webauthn4j to 0.20.0

### LINKED ISSUE #12311: Update webauthn4j to 0.20.0
### Description

Update webauthn4j to 0.20.0

    Yesterday webauthn4j release a new version which contains a couple of
    bug fixes plus CVE updates for its dependencies:

    Breaking changes
        - Add EdDSA support #662
        - Correct AuthenticationAlgorithm(0x0011) value #657

    Dependency Upgrades
        - Bump spring-boot-dependencies from 2.6.7 to 2.7.0 #661
        - Bump jacksonVersion from 2.13.2 to 2.13.3 #660
        - Bump kerby-asn1 from 2.0.1 to 2.0.2 #659
    Bump checker-qual from 3.21.4 to 3.22.0 #654¨

## More details
- https://github.com/webauthn4j/webauthn4j/releases/tag/0.20.0.RELEASE

### PR DESCRIPTION
**Pull request by abstractj**
Yesterday, webauthn4j released a new version which contains a couple of
bug fixes plus CVE updates for its dependencies:

Breaking changes
    - Add EdDSA support #662
    - Correct AuthenticationAlgorithm(0x0011) value #657

Dependency Upgrades
    - Bump spring-boot-dependencies from 2.6.7 to 2.7.0 #661
    - Bump jacksonVersion from 2.13.2 to 2.13.3 #660
    - Bump kerby-asn1 from 2.0.1 to 2.0.2 #659
Bump checker-qual from 3.21.4 to 3.22.0 #654

Resolves #12311

### CODE REVIEW

**Comment by sguilhen** [comment]
@martin-kanis I believe this need to be productized, latest version I found for `webauthn4j-core` and `webauthn4j-util` in PNC is 0.19.3.

**Comment by martin-kanis** [comment]
@sguilhen Eventually it needs to be productized. When, depends on a requirement to backport it in a micro release.

**Review by sguilhen — APPROVED** [review]

**Comment by mposolda** [comment]
@martin-kanis Is it ok to merge this to keycloak main even if webauthn4j 0.20.0 is not yet productized in PNC? Please approve this PR if it is ok.

@mabartos Can you please doublecheck if upgrade of webauthn4j does not break anything? I see that GH action was not executed with this PR since there is no change in webauthn related part (just dependency updates in pom.xml etc), however this chance can surely have impact on the WebAuthn related functionality and hence I guess pipeline run may be needed before this PR is merged?

**Comment by mabartos** [comment]
@mposolda Sure, thanks for notifying me. I'll execute the pipeline and let you know about the results.

**Review by martin-kanis — APPROVED** [review]
Approving. Just I would like to notify @pskopek that webauthn4j 0.20.0.RELEASE needs to be productized when this finds its path to downstream.

**Comment by mposolda** [comment]
@martin-kanis @mabartos Thanks!

**Review by mabartos — APPROVED** [review]
Pipeline tests passed (present failures are not related to this PR).
LGTM

https://keycloak-jenkins.com/job/universal-test-pipeline-server/1613/testReport/

**Comment by mposolda** [comment]
@mabartos Cool, Thanks a lot for testing this!

**Review by mposolda — APPROVED** [review]

**Pull request merged**
